sections/map_elements.tex

```diff
@@ -41,7 +41,8 @@ \subsection*{Map Tile Anatomy\index{Map Tile}}
     \item[\textbf{1.}] Empty Field
     \item[\textbf{2.}] Location
     \item[\textbf{3.}] Artifact Symbol
-    \item[\textbf{4.}] Blocked Field
+    \columnbreak
+    \item[\textbf{4.}] Blocked Field,\\ name of the tile
```

Comment:
```suggestion
    \item[\textbf{4.}] Blocked Field,\\ tile name like N1
```
Since the picture doesn't show the actual name, how about something like this?

Reply:
I actually was thinking of replacing the image with one which shows the name clearly